common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -141,6 +141,11 @@ class DB(contextWrapper: ContextWrapper) {
     contextWrapper.dbContext.run(action)
   }
 
+  val curatedStatuses = List(RecipeStatusVerified, RecipeStatusCurated, RecipeStatusFinalised, RecipeStatusPendingCuration, RecipeStatusPendingVerification, RecipeStatusPendingFinalisation).map(_.name)
```

Comment:
Hi @tomrf1 

Now that you mention it, you are *perfectly* right that `RecipeStatusPendingCuration` should be excluded from the list. Because in this state that have never been submitted and are simply coming from `Ready` status. This is my mistake, I should have thought about that. 

That having been said, I don't see any problem with `RecipeStatusPendingVerification` and `RecipeStatusPendingFinalisation`.